source/connect/connection-options.txt

```diff
@@ -0,0 +1,286 @@
+.. _cpp-connection-options:
+
+==========================
+Specify Connection Options
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, configure
+
+Overview
+--------
+
+This page describes the MongoDB connection and authentication options
+available in the {+driver-short+}. You can configure your connection by using
+options in the connection URI.
+
+.. note::
+
+   You can set most connection options by using either the connection URI or the
+   ``client_options`` parameter. However, some options support only one of these
+   methods. You might need to use both a connection URI and the ``client_options``
+   parameter to set all the options that you need.
+
+.. _cpp-connection-uri:
+
+Using the Connection URI
+------------------------
+
+When you construct a ``mongocxx::client`` object, you can pass in a ``mongocxx::uri``
+object that represents your connection URI. This connection URI can include
+connection options as ``<name>=<value>`` pairs. In the following example,
+the connection URI contains the ``tls`` option with a value of ``true`` and the
+``tlsCertificateKeyFile`` option with a value of ``path/to/file.pem``:
+
+.. literalinclude:: /includes/connect/connection-options.cpp
+   :language: cpp
+   :copyable: true
+
```

Comment:
S: you can add a section here (maybe called "Using an Options Parameter") about how to specify options by setting fields of a `mongocxx::client::options` object, and passing that object to the `mongocxx:client` constructor